services/docker-compose.yml

```diff
@@ -590,9 +590,6 @@ services:
       DIRECTOR_V2_HOST: ${DIRECTOR_V2_HOST}
       DIRECTOR_V2_PORT: ${DIRECTOR_V2_PORT}
 
-      # WEBSERVER_DIRECTOR
-      DIRECTOR_HOST: ${DIRECTOR_HOST}
```

Comment:
Are these 2 ENV now obsolete and need to be removed from `repo.config`?

Reply:
@YuryHrytsuk No, they are not obsolete. That is what I tried to explain in the description above. The are used only in two services (director-v2 and catalog) but they were exposed to five!